docs/en/infraops/page_header.html

```diff
@@ -0,0 +1,5 @@
+You are looking at documentation for an older release.
+Not what you want?
+In the current release, the Infrastructure monitoring guide has been replaced by two documents, the
+<a href="https://www.elastic.co/guide/en/logs/guide/current/index.html"> Logs monitoring guide</a>
+and the <a href="https://www.elastic.co/guide/en/metrics/guide/current/index.html"> Metrics monitoring guide</a>.
```

Comment:
I'd avoid referencing current and be more explicit. Something like:

> Starting in version 7.5, see the Logs monitoring guide and the Metrics monitoring guide for information about infrastructure monitoring.

With the appropriate links, of course. 



>

Reply:
@debadair Agreed, except I think the links probably still need to be to the current versions? If the reader finds the wrong docs, when we direct them to the right docs, the most helpful version to take them to initially is probably the current version? From there they can navigate to another version if they want to.

I'm just checking this because you seem to be saying that I need to change the links here too. At present, they do go to the current version. Am I missing something?